tools/header-dump/HeaderDump.hs

```diff
@@ -374,7 +372,7 @@ withBlockHeaders logger config inner = do
         initializePayloadDb v pdb
         liftIO $ logg Info "start traversing block headers"
         liftIO $ logg Info $ "header validation: " <> sshow (_configValidate config)
-        withChainDbsConcurrent rdb v cids (_configValidate config) start end $ inner pdb . void
+        withChainDbs rdb v cids (_configValidate config) start end $ inner pdb . void
```

Comment:
So is this single threaded now?

Reply:
Yes, but performance seemed still fine. Well, actually performance never really was that great :-) I think, this has bottlenecks elsewhere.

Also, I think, I am the only one using this code in personal scripts.